schema-bridge.graphql

```diff
@@ -310,6 +561,45 @@ type Pool @entity {
   hourlySnapshots: [PoolHourlySnapshot!]! @derivedFrom(field: "pool")
 }
 
+type PoolRoute @entity {
+  # Still thinking about how to make this ID in a way that allows routes to be linked among chains 
```

Comment:
could it be linked by inputToken and crossToken? chain `A` crossToken.token == chain `B` inputToken and vice versa

Reply:
That would be ideal. I fear that we might not be able to always know the address of the token on the other chains. 

Although I’m assumig that we are by making `CrosschainToken.token` a non-nullable field 😅

Reply:
I was hoping to be able to have the same ID for the same route in all the different deployments, but the ways I can think of achieving that are either not simple or would make it impossible on one of the sides of the chain to load a given route from a mapping knowing only about the pool and the chains involved.

Will leave it as is. Identifying same routes by the relationship of their inputToken and crossToken should work.